Xamarin.Forms.Platform.WPF/CellControl.cs

```diff
@@ -93,11 +93,11 @@ protected override void OnRenderSizeChanged(SizeChangedInfo sizeInfo)
 				{
 					foreach (var child in vc.LogicalChildren)
 					{
-						if (child is Layout layout)
+						if (child is View view)
```

Comment:
for case when child is not layout (e.g. label) with horizontal options = ...AndExpand